File: client/js/mitarbeiter.js
import { getData } from "./script.js";

export class Mitarbeiter {
    constructor(id, vorname, nachname, geburtsdatum, geschlecht, qualifiziert, notiz) {
        this.txt_id = id;
        this.vorname = vorname;
        this.nachname = nachname;
        this.geburtsdatum = geburtsdatum;
        this.geschlecht = geschlecht;
        this.qualifiziert = qualifiziert;
        this.notiz = notiz;
    }
}

File: WebApplication1/WebApplication1/Models/Mitarbeiter.cs
using Microsoft.EntityFrameworkCore.Metadata.Internal;
using WebApplication1.Services;
using System.ComponentModel.DataAnnotations;

namespace WebApplication1.Models
{
    public class Mitarbeiter
    {

        [Key]
        public int Id { get; set; }

        public string? Vorname { get; set; }
        public string? Nachname { get; set; }
        public DateTime Geburtsdatum { get; set; }
        public int Geschlecht { get; set; }
        public bool Qualifiziert { get; set; }
        public string? Notiz { get; set; }

    }
}
